apps/devhub/tests/test_views_validation.py

```diff
@@ -89,6 +90,12 @@ def test_dupe_uuid(self, flag_is_active):
               'type': 'error', 'fatal': True}])
         eq_(data['validation']['ending_tier'], 1)
 
+    def test_too_long_uuid(self):
+        """An add-on uuid must be 64chars at most, see bug 1201176."""
+        with self.assertRaises(forms.ValidationError,
+                               msg='UUID must be less than 64chars'):
+            parse_addon(self.get_upload('extension-too-long-guid.xpi'))
```

Comment:
Any chance you could try this using `files.utils.make_xpi` (as in https://github.com/mozilla/olympia/blob/master/apps/zadmin/tests/test_tasks.py#L46) rather than creating a file fixture? Fixtures can be kind of opaque, and I'd much rather be able to look at exactly what this is testing rather than locate and unzip an XPI.